tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.txt

```diff
@@ -0,0 +1,38 @@
+<Configuration> 
+ <TestRoot>..\..</TestRoot> 
+ <TargetArchitecture>x64</TargetArchitecture> 
+ <ScriptRootDir>$(TestRoot)\Scripts</ScriptRootDir> 
+ <DebuggeeRootDir>$(TestRoot)\Debuggees</DebuggeeRootDir> 
+ <DebuggeeSourceRoot>$(DebuggeeRootDir)\Source</DebuggeeSourceRoot>
+ <RuntimeRoot>##Insert_Runtime_Root##</RuntimeRoot>
+ <NugetCacheRoot>##Insert_Nuget_Cache_Root##</NugetCacheRoot>
+ <Options> 
```

Comment:
You can get rid of the Options and Option tags as there is only 1 option here. It removes a concept that someone would need to understand when trying to figure out what the config file does.